datasets/aeslc/README.md

```diff
@@ -3,6 +3,18 @@ languages:
 - en
 paperswithcode_id: aeslc
 pretty_name: AESLC
+task_categories:
+- text-classification
+arxiv: "1906.03497"
```

Comment:
the arxiv tag is not supported, see [the dataset tagging app](https://huggingface.co/spaces/huggingface/datasets-tagging)
```suggestion
```